app/jobs/reports/base_report.rb

```diff
@@ -27,22 +41,8 @@ def end_of_today
       Time.zone.now.end_of_day
     end
 
-    def report_timeout
```

Comment:
we never overrode this timeout in any instances, so I just removed it by inlining it